### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by Lawouach**
Minor changes so that the test pass.

### CODE REVIEW

File: src/test/java/com/atomist/rug/cli/AbstractCommandTest.java [hunk 1]
```diff
@@ -112,6 +112,7 @@ protected void assertSuccess(String requiredContents, String... tokens) throws E
             commandLine.add(config.getCanonicalPath());
             commandLine.add("-X");
             commandLine.add("-t");
+            commandLine.add("-l");
```
Comment (cdupuis) [inline src/test/java/com/atomist/rug/cli/AbstractCommandTest.java]: That won't work for all commands

**Comment by cdupuis** [comment]
Those changes will make half of the tests fail.

File: src/test/java/com/atomist/rug/cli/AbstractCommandTest.java (hunk 1 above)
Comment (Lawouach) [inline src/test/java/com/atomist/rug/cli/AbstractCommandTest.java]: Oh. That's unfortunate. As it stands, tests have been failing without it. I will add that flag only to tests that were failing before the change, is that okay?

**Comment by Lawouach** [comment]
I went for a more surgical option. I also ran `mvn clean install` and the whole test suite passed locally (which was a first). Let's see what the automated build has to say.

**Comment by cdupuis** [comment]
LGMT. Thanks

**Pull request merged**